### DISCUSSION TITLE
For iOS and Linux support any plan?

### DISCUSSION THREAD

**Discussion by dipeshSam** [Akascape/py-window-styles#18] [General]
I visited this repository. It's really good work that you've combined all the options available to manipulate Windows window object.

What's about **MacOS** and **Linux**?? Are you working on??? Or, is it just **not** possible in these operating systems???

**Comment by Akascape** [comment]
@dipeshSam It's currently only for windows because there are different window customisation developed for it, and this package just activates those internal functions through ctypes. You can't expect those exact features in other OS because they are only developed for windows.

Different OS have different window managers. For windows it's windll.

For macos or linux, I don't know which library functions to tweak, or what features they provide, moreover finding that particular documentation is also very difficult.

So, no plan from my side.

**Comment by dipeshSam** [comment]
@Akascape Okay, got it.

Your work for Windows is also appreciable.
Keep it up.

Best regards.
